### PR TITLE
Migrate diskusage to 1.0.0

### PR DESCRIPTION
**Pull request by acid-chicken**
***DEPENDS #3498***

### CODE REVIEW

**Review by AyaMorisawa — APPROVED** [review]

**Comment by syuilo** [comment]
🙏

**Comment by acid-chicken** [comment]
Note: Please don't merge before #3498 merges, but it's a trivial problem.

**Pull request merged**
